tensorflow_lite_support/python/task/vision/pybinds/_pywrap_image_embedder.cc

```diff
@@ -88,7 +96,15 @@ PYBIND11_MODULE(_pywrap_image_embedder, m) {
 
              ASSIGN_OR_RETURN(std::unique_ptr<FrameBuffer> frame_buffer,
                               CreateFrameBufferFromImageData(image_data));
-             return self.Embed(*frame_buffer, vision_bounding_box);
+             ASSIGN_OR_RETURN(EmbeddingResult vision_embedding_result,
+                              self.Embed(*frame_buffer, vision_bounding_box));
+
+             // Convert from vision::EmbeddingResult to
+             // processor::EmbeddingResult
+             processor::EmbeddingResult embedding_result;
+               embedding_result.ParseFromString(
+                       vision_embedding_result.SerializeAsString());
+             return embedding_result;
            })
       .def("get_embedding_by_index",
```

Comment:
You also need to update get_embedding_by_index and cosine_similarity to return and take the processor::Embedding value rather than the current vision::Embedding.